BOILERPLATE_README.md

```diff
@@ -22,11 +22,11 @@
 
 ## 🚧 Dependencies
 
-- Node.js (`10.14.2`)
-- NPM (`6.4.1`)
-- Elixir (`1.8.1`)
-- Erlang (`21.3.3`)
-- PostgreSQL (`~10.0`)
+- Node.js (`~> 10.14`)
+- NPM (`~> 6.9`)
```

Comment:
Node 10 is bundled with `6.4.1`, why are we upgrading to `6.9.0`?

Furthermore, we should probably upgrade to the latest `10.15.3`.

Reply:
+1 for upgrading to latest `10.15.3`

Reply:
> Node 10 is bundled with 6.4.1, why are we upgrading to 6.9.0?

Because we can do it easily 😄 Is there a reason we should _not_ do?

> Furthermore, we should probably upgrade to the latest 10.15.3.

Unfortunately, the latest `nodejs` and `npm` version on our Alpine version is `10.14.2-r0` (even if `10.15.3-r0` [seems](https://pkgs.alpinelinux.org/package/edge/main/x86/npm) to be available; I don’t know Alpine enough to upgrade to it.)

Reply:
`10.15.3-r0` only is available on the `edge` branch, but `elixir:1.8.1-alpine` is based on Alpine `3.9`

https://pkgs.alpinelinux.org/package/v3.9/main/x86/nodejs